src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,23 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <UserInfo user={post.user} />
```

Comment:
user can be undefined, let's add the check
```suggestion
      {post.user && <UserInfo user={post.user} />}
```